cards.c

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <string.h>
+#include <time.h>
+#include <stdlib.h>
+
+enum suits{"of spades","of hearts","of diamonds","of clubs"};
```

Comment:
so your first error is that each of these should be a single word variable. so it should be written as something like: 
``` C
enum suits { SPADES, DIAMOND, CLUB, HEARTS }
```